Scenarios/Scenario.swift

```diff
@@ -17,13 +17,22 @@ public final class Scenario: Preparable, Actionable, ScenarioBuilder {
   private let commitFunc: CommitFunc
   private var stepDescriptions: [StepMetadata] = []
 
-  public init(_ name: String, file: String = #file, line: UInt = #line, commit: @escaping CommitFunc = quick_it) {
+  public init(_ name: String, file: String = #file, line: UInt = #line, commit: @escaping CommitFunc) {
     self.name = name
     self.file = file
     self.line = line
     self.commitFunc = commit
   }
 
+  public convenience init(_ name: String, file: String = #file, line: UInt = #line) {
```

Comment:
Unfortunately the default argument doesn't actually compile under Swift 4: turns out you can't use something that's `private` as a default value in something that's `public` (I think this is an ABI stability fix). Using a convenience initialiser means the implementation is private, and is free to reference private members.